profiles/docker/deploy

```diff
@@ -17,29 +36,31 @@ fi
 
 maybe_verb=$(basename "$0" | cut -d- -f3)
 if test -z "$maybe_verb"; then
-  usage
+  color e "Unable to find local executor"
   exit 1
 fi
 verb="$maybe_verb"
 
-cat << EOF >.hub-toolbox
-#!/bin/bash -e
+run_toolbox_script="$HUB_WORKDIR/.hub/.hub-toolbox"
+cat << EOF > "$run_toolbox_script"
+#!/bin/sh -e
 
 hubctl="\$(which hubctl 2>/dev/null || true)"
 if test -z "\$hubctl"; then
   hubctl="\$(which hub 2>/dev/null || true)"
 fi
 
-if test -e ".env"; then
+if test -e "\$HUB_WORKDIR/.env"; then
+if test -e "\$HUB_WORKDIR/.env"; then
     set -a
-    source .env
+    source \$HUB_WORKDIR/.env
```

Comment:
Instead, there should be `. \$HUB_WORKDIR/.env`

Reply:
IMHO `source` it is not correct source .env file. Because `.env` doesn't have to be shell compatible. We must source shell compatible outputs produced by `dotenv` as the following stanza

```bash
eval "$(dotenv -f "$HUB_WORKDIR/.env" "export")"
```

Yet I would favour in general to use `source` instead of `.`